addons/website/static/src/snippets/s_media_list/001.scss

```diff
@@ -0,0 +1,13 @@
+.s_media_list[data-vcss="001"] {
+    .s_media_list_body {
+        padding: $grid-gutter-width;
+    }
+    .s_media_list_img_wrapper {
+        align-self: stretch;
+        .s_media_list_img {
+            height: 100%;
+            width: 100%;
+            object-fit: cover;
+        }
+    }
```

Comment:
Useless css nesting here ;)
But in fact, I feel like all those rules are useless: they have xml classes equivalent, so we should use them.